.travis.yml

```diff
@@ -1,5 +1,4 @@
 language: ruby
-cache: bundler
```

Comment:
@cbeer been running into issues with this too. Have you looked at this? http://docs.travis-ci.com/user/caching/#Clearing-Caches